Tests/Weaver/Realm.Fody.Tests/Realm.Fody.Tests.csproj

```diff
@@ -7,6 +7,18 @@
     <GenerateProgramFile>false</GenerateProgramFile>
   </PropertyGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Include="..\..\Realm.Tests\EmbeddedResources\nunit3-junit.xslt">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </EmbeddedResource>
+
+    <Compile Include="..\..\Realm.Tests\TransformHelpers.cs" Link="TransformHelpers.cs" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <LangVersion>8.0</LangVersion>
```

Comment:
This was added to allow C# 8 "using" syntax in fody since it targets also net472, which only supports C# 7.3 unless "forced" to use some parts of C# 8.